src/api/bank-accounts/bank-account-requests.md

```diff
@@ -0,0 +1,107 @@
+---
+layout:  default
+grand_parent: API Reference
+parent: Bank Accounts
+title: Bank Account Requests
+permalink: /api/bank-account-requests
+---
+
+# Bank Account Requests
+{:.no_toc}
+
+Bank Account Requests facilitate user authorization of access to bank accounts. Typically the authorization will follow an OAuth flow where the user is redirected to a third-party system to authenticate and select one or more bank accounts. After a bank account request has been completed, one or more Centrapay Bank Account resources may be created and long-lived access tokens may be stored against those Bank Accounts.
+
+## Contents
+{:.no_toc .text-delta}
+
+* TOC
+{:toc}
+
+## Models
+
+### Bank Account Request
+{% h4 Mandatory Fields %}
```

Comment:
I noticed here it says `Mandatory Fields` and below it says `Required Fields` - maybe these should use the same term?

Reply:
Mandory fields is for the model. I'll change required fields to 'fields' as we're doing optional ones with a tag now